interfaces/builtin/mpris.go

```diff
@@ -189,18 +187,23 @@ func (iface *MprisInterface) ConnectedPlugSnippet(plug *interfaces.Plug, slot *i
 func (iface *MprisInterface) PermanentSlotSnippet(slot *interfaces.Slot, securitySystem interfaces.SecuritySystem) ([]byte, error) {
 	switch securitySystem {
 	case interfaces.SecurityAppArmor:
-		snippet := mprisPermanentSlotAppArmor
+		name, err := iface.GetName(slot.Attrs)
```

Comment:
This should be a private method since nothing apart from this module needs to call it.